blog/_posts/2024-06-25-history-ele-raid-buffs.md

```diff
@@ -192,7 +198,8 @@ Elemental Shamans also received [Skyfury Totem](https://wowpedia.fandom.com/wiki
 Shamans also received [Wind Rush Totem](https://wowpedia.fandom.com/wiki/Wind_Rush_Totem) in Legion.
 This new utility spell provided a movement speed buff to the group similar to Druid's [Stampeding Roar](https://wowpedia.fandom.com/wiki/Stampeding_Roar) but with positional limitations.
 In Dragonflight we once again received [Mana Spring](https://wowpedia.fandom.com/wiki/Mana_Spring) (first as a totem, then as a buff), but its effectiveness was severely dampened to the point where most don't consider it a valuable.
-Dragonflight also brought several changes to Ancestral Guidance - which was at first a powerful group heal, especially in AOE situations - but it was heavily nerfed in 10.1.5 and left a shell of its former self.
+Less notably, Dragonflight brought back several other totems with minor usefulness: [Tremor Totem](https://www.wowhead.com/spell=8143/tremor-totem), [Stoneskin Totem](https://www.wowhead.com/spell=78222/stoneskin-totem), [Tranquil Air Totem](https://www.wowhead.com/spell=383019/tranquil-air-totem) (mainly a PVP ability), and [Poison Cleansing Totem](https://www.wowhead.com/spell=383015/poison-cleansing).
```

Comment:
Also forgot to mention these from Dragonflight since they're pretty much all super minor